### ISSUE TITLE
Implement modules system

### ISSUE THREAD

**Issue by F1uctus** [F1uctus/Axion#13]
Modules should be organized by nesting directories.

Definitions from one module should have straight accessibility from other module's parts.
e.g: examine the hierarchy:
```
app1
- frontend
  - button.ax
  - list-view.ax
- backend
  - converter.ax
  - parser.ax
```

Here, parts of module `frontend` should have access to each other without imports.
Same goes for `backend`.

**Comment by F1uctus** [comment]
Modules & Projects system is partly implemented now. Now I'm moving to the `import` statements implementation.
